pmlc/dialect/pxa/transforms/BUILD

```diff
@@ -23,6 +26,7 @@ plaidml_cc_library(
         "//pmlc/dialect/pxa/ir",
         "//pmlc/dialect/tile/ir",
         "//pmlc/dialect/eltwise/ir",
+        "//pmlc/target/x86",
```

Comment:
I think we should consider how we're going to integrate the components.

We have the generic stencil pass, but it should be configured with a specific costing model. The stencil pass doesn't need to know about the costing model directly, it could take a `std::function` or even a class with virtual methods on it. That way we could have the `x86` pipeline decide that it wants to use the stencil pass with the x86-specific heatmap applied.

Reply:
We should. Let's talk about it. The heatmap is a standalone component, I think.

Reply:
OK, I think we should configure the stencil pass with a cost model. You could have it be in the form of a function that takes in a tile and returns a cost.

The big deal is that I don't think we should have the PXA dialect depend on the x86 target. It should be the other way around.

Reply:
Sure! I'll start prototyping things as soon as this gets in. This is just putting the heatmap in. I think we need this no matter what and this could go in... Small checkins is good...